ethtool.go

```diff
@@ -1024,6 +1032,20 @@ func (e *Ethtool) Close() {
 	unix.Close(e.fd)
 }
 
+// Identity the nic with blink duration, if not specify blink infinity
+func (e *Ethtool) Identity(intf string, duration *time.Duration) error {
+	var dur uint32
```

Comment:
could we use a default duration instead of zero ?

Reply:
how about 60s